contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
The other thing I would recommend is changing `ceph-deploy install --dev master` to something stable like `ceph-deploy install --release hammer`